### ISSUE TITLE
Makeinstall.bat does not copy extracted .DMG/chunklists

### ISSUE THREAD

**Issue by kweinberg34** [corpnewt/gibMacOS#259]
Attempting to make a USB installer in Windows 10 for 001-68446 - 10.15.7 macOS Catalina. GibMacOS retrieves the files, no problem. MakeInstall.bat (options- select disk 2 [Sandisk 16 GB], o- opencore) creates 2 partitions: H:\ (196MB, FAT32) and a hidden, unformatted, unlabeled 14.4GB partition. Booting structures are created successfully to H:\. AppleDiagnostics.DMG and BaseSystem.DMG and associated chunklists are MIA. BaseSystem.DMG is way too big to fit in the H:\ partition. BTW: I have these files extracted to "...\gibMacOS\gibMacOS-master\macOS Downloads\publicrelease\001-68446 - 10.15.7 macOS Catalina\RecoveryHDMetaDmg\RecoveryHDMeta"

I've booted the USB drive and it gets through the boot menu, I ask to boot MacOS, I get about a page of text that gives me hope and then the screen goes black and the computer reboots.

Is this an issue with that second partition? ANY assistance would be sincerely appreciated, folks!

**Comment by tianyehuakai** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by kweinberg34** [comment]
Right- I understand that. I only added that in case of a potential workaround. I just don't know what it DOES copy, and where it's supposed to put them. In other words, I don't know if everything has been working correctly and the boot to the recovery image fails because my config.plist/kexts/drivers, etc. are all ate up or if requisite files are missing from the USB (somewhere... It sure wouldn't be the H:\ drive...)

**Comment by kweinberg34** [comment]
Thanks so much, CorpNewt!
